src/github3/__about__.py

```diff
@@ -5,11 +5,8 @@
 __author_email__ = "graffatcolmingov@gmail.com"
 __license__ = "Modified BSD"
 __copyright__ = "Copyright 2012-2022 Ian Stapleton Cordasco"
-__version__ = "3.2.0"
-__version_info__ = tuple(
-    int(i) for i in __version__.split(".") if i.isdigit()
-)
 __url__ = "https://github3.readthedocs.io"
+from ._version import __version__, __version_info__
```

Comment:
I don't see this _version file in your PR. Instead of having a separate file, can't we use pyproject.toml and use importlib_metadata in the code to fetch it?

Reply:
Hatch generates the version file at build time from the template in `pyproject.toml`; that stems from the tools that do version bumps from tags, if you use the whole suite of hatch tools (which you _can_ but don't need to). So, the version generates the file, not the other way around. The file is in `.gitignore` intentionally.